src/test/java/com/exasol/rls/administration/scripts/ScriptsSqlManager.java

```diff
@@ -0,0 +1,41 @@
+package com.exasol.rls.administration.scripts;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class ScriptsSqlManager {
```

Comment:
Please turn into a class with instance methods.
Initialize with a connection injected via constructor.
Rename to `SqlTestSetupManager`.

Reply:
done